src/builder/app.ts

```diff
@@ -4,6 +4,9 @@ import {
   IResponseBase,
   IGenerateCodeResponse,
 } from './types';
+import { checkNodeInstallation } from './utils/nodeUtil';
+import { validateUserInput } from './utils/validationUtil';
+import { logStage } from './utils/loggingUtil';
 
```

Comment:
@Toowiredd Thank you for the PR. The commit is missing the validation utils you added.